tutorbackup/patches/k8s-jobs

```diff
@@ -179,6 +187,10 @@ spec:
                   value: '{{ MYSQL_ROOT_USERNAME }}'
                 - name: MYSQL_ROOT_PASSWORD
                   value: '{{ MYSQL_ROOT_PASSWORD }}'
+                {% if BACKUP_MYSQL_DATABASES %}
+                - name: MYSQL_DATABASES
+                  value: '{{ BACKUP_MYSQL_DATABASES | join(" ") }}'
```

Comment:
same as above

Reply:
I saw those quotes weren't translated to the settings, but anyway I removed them

Reply:
It's still there.